public/controllers/agent/wazuh-config/index.ts

```diff
@@ -0,0 +1,373 @@
+const architectureButtons = [
+  {
+    id: 'i386',
+    label: 'i386',
+  },
+  {
+    id: 'x86_64',
+    label: 'x86_64',
+  },
+  {
+    id: 'armhf',
+    label: 'armhf',
+  },
+  {
+    id: 'aarch64',
+    label: 'aarch64',
+  },
+];
+
+const architectureButtonsWithPPC64LE = [
+  {
+    id: 'i386',
+    label: 'i386',
+  },
+  {
+    id: 'x86_64',
+    label: 'x86_64',
+  },
+  {
+    id: 'armhf',
+    label: 'armhf',
+  },
+  {
+    id: 'aarch64',
+    label: 'aarch64',
+  },
+  {
+    id: 'powerpc',
+    label: 'PowerPC',
+  },
+];
+
+
+const architectureButtonsi386 = [
+  {
+    id: 'i386',
+    label: 'i386',
+  }
+];
+
+const architecturei386Andx86_64 = [
+  {
+    id: 'i386',
+    label: 'i386',
+  },
+  {
+    id: 'x86_64',
+    label: 'x86_64',
+  },
+];
+
+const architectureButtonsSolaris = [
+  {
+    id: 'i386',
+    label: 'i386',
+  },
+  {
+    id: 'sparc',
+    label: 'SPARC',
+  }
+];
+
+const architectureButtonsMacos = [
+  {
+    id: 'intel',
+    label: 'Intel'
+  },
+  {
+    id: 'applesilicon',
+    label: 'Apple Silicon'
+  }
+]
+
+const architectureButtonsOpenSuse = [
+  {
+    id: 'x86_64',
+    label: 'x86_64',
+  },
+  {
+    id: 'ARM64',
+    label: 'ARM64',
+  }
+];
+
+const architectureButtonsAix = [
+  {
+    id: 'powerpc',
+    label: 'PowerPC',
+  }
+];
+
+const architectureButtonsHpUx = [
+  {
+    id: 'itanium2',
+    label: 'Itanium2',
+  }
+];
+
+const versionButtonAmazonLinux = [
+  {
+    id: 'amazonlinux1',
+    label: 'Amazon Linux 1',
+  },
+  {
+    id: 'amazonlinux2',
+    label: 'Amazon Linux 2',
+  },
+  {
+    id: 'amazonlinux2022',
+    label: 'Amazon Linux 2022',
+  }
+]
+
+const versionButtonsRedHat = [
+  {
+    id: 'redhat5',
+    label: 'Red Hat 5',
+  },
+  {
+    id: 'redhat6',
+    label: 'Red Hat 6',
+  },
+  {
+    id: 'redhat7',
+    label: 'Red Hat 7 or higher',
+  },
+];
+
+const versionButtonsCentos = [
+  {
+    id: 'centos5',
+    label: 'Centos 5',
+  },
+  {
+    id: 'centos6',
+    label: 'Centos 6',
+  },
+  {
+    id: 'centos7',
+    label: 'Centos 7 or higher',
+  },
+];
+
+const versionButtonsDebian = [
+  {
+    id: 'debian7',
+    label: 'Debian 7',
+  },
+  {
+    id: 'debian8',
+    label: 'Debian 8',
+  },
+  {
+    id: 'debian9',
+    label: 'Debian 9',
+  },
+  {
+    id: 'debian10',
+    label: 'Debian 10 or higher',
+  }
+];
+
+const versionButtonFedora = [
+  {
+    id: '22',
+    label: '22 or later'
+  }
+]
+
+const versionButtonsUbuntu = [
+  {
+    id: 'ubuntu14',
+    label: 'Ubuntu 14',
+  },
+  {
+    id: 'ubuntu15',
+    label: 'Ubuntu 15 or higher',
+  },
+  {
+    id: 'ubuntu16',
+    label: 'Ubuntu 16 or higher',
+  }
+];
```

Comment:
I think it would be ubuntu 14, ubuntu 15 or ubuntu 16 or higher,

or is ubuntu 16 left over since it also complies to be higher than ubuntu 15?

Reply:
Fixed